byway/src/main/java/com/google/sps/servlets/PlacesServlet.java

```diff
@@ -39,6 +41,43 @@
   /** {@link Type} of an {@link ArrayList} containing {@link String}, for gson decoding. */
   private static final Type ARRAYLIST_STRING = new TypeToken<ArrayList<String>>() {}.getType();
 
+  /**
+   * Formatted to be all lowercase, delimited by an underscore. Follows the format used for
+   * PlaceType objects.
+   */
+  private static final ImmutableSet<String> NON_INTERESTS =
```

Comment:
Can you name this something slightly more descriptive? hard to know what a non-interest is in this context. Something like NON_INTEREST_PLACETYPES or something that otherwise describes what these values are in the larger context.

Reply:
Updated both name and its description